src/localplayer.cpp

```diff
@@ -631,7 +640,7 @@ void LocalPlayer::applyControl(float dtime)
 					speedV.Y = movement_speed_walk;
 			}
 		}
-		else if(m_can_jump || (control.sneak && m_sneak_ladder_detected))
+		else if(m_can_jump)
 		{
```

Comment:
Looking at the code, there was no point fixing this here but not in surrounding code, there are so many code style issues they need to be done in another commit.